### PR TITLE
Fix: Overlapping menu on sign up

### LINKED ISSUE #900: Overlapping Menu and Systers Logo when resizing the browser
## Description

When running locally, if we resize the browser then Horizontal Menu overlaps on Systers Logo.

### Update [Required]
- [ ] Adaptive menu which changes when the browser size is less than a specific value
- [ ] Make it consistent for all the pages

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by mvortizr**
# Description
The navbar was overlapping with the logo and the responsive view of the navbar was unaligned.

Fixes #900

# Type of Change:
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
Firefox and Chrome with the Dev Tools

systers1
systers2

gif_vms

# Checklist:
**Delete irrelevant options.**
- [x] Adaptive menu which changes when the browser size is less than a specific value
- [x] Make it consistent for all the pages

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Monal5031) [inline .vscode/settings.json]: Please remove this file

File: vms/administrator/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```
Comment (Monal5031) [inline vms/administrator/migrations/0001_initial.py]: No... Please remove migrations

File: vms/event/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
```
Comment (Monal5031) [inline vms/event/migrations/0001_initial.py]: Same here

File: vms/vms/static/vms/css/style.css [hunk 4]
```diff
@@ -65,7 +66,7 @@ body {
   top:20px;
   font-size: 16px;
   font-weight: bolder;
-  font-family: 'Cairo';
+  font-family: 'Cairo';
```
Comment (Monal5031) [inline vms/vms/static/vms/css/style.css]: Please remove this extra space as well

File: vms/vms/static/vms/css/style.css [hunk 5]
```diff
@@ -88,20 +90,24 @@ body {
   background-color: #288F79;
 }
 
-#signup-btn {
+#signup-btn, #signup-admin-btn{
   background-color: #8d2165;
-  right: 20%;
 }
 
+/*#signup-btn {
```
Comment (Monal5031) [inline vms/vms/static/vms/css/style.css]: Remove this commented code as well

File: vms/vms/static/vms/css/style.css [hunk 6]
```diff
@@ -88,20 +90,24 @@ body {
...
+  right: 20%;
+} */
+
 
 #signup-btn:hover, #signup-admin-btn:hover {
   background-color: #690344;
 }
 
-#signup-admin-btn {
+/*#signup-admin-btn {
```
Comment (Monal5031) [inline vms/vms/static/vms/css/style.css]: Same here

Comment (marylicious) [reply vms/vms/static/vms/css/style.css]: > Do not commit migration files
sorry I will fix itt

**Comment by marylicious** [comment]
I think it's done, should I do rebase to eliminate the double commit?

**Comment by marylicious** [comment]
ok, done.  I did the modification and squashed the commits

**Comment by marylicious** [comment]
I think it's done now.

**Comment by snitin315** [comment]
I will test it locally in a while.

**Comment by snitin315** [comment]
@marylicious can you provide a working GIF? It's not that clear from screenshots.

**Comment by saurabhkanswal** [comment]
@mvortizr / @marylicious can you please provide GIF. so @snitin315  can approve your pull request.
thanks.

**Comment by mvortizr** [comment]
off I forgot, I will do it tomorrow

**Comment by mvortizr** [comment]
I'm on another computer and the installation of the things I need took longer than expected, I think I will do this by tomorrow.

**Comment by saurabhkanswal** [comment]
> I'm on another computer and the installation of the things I need took longer than expected, I think I will do this by tomorrow.

not a problem, take your time😊

**Comment by mvortizr** [comment]
done, is that gif ok?

**Pull request merged**